docs/roman/ramp_fitting/description.rst

```diff
@@ -193,6 +193,114 @@ Upon successful completion of this step, the status attribute ramp_fit will be s
 to "COMPLETE".
 
 
+Jump Detection
+==============
+
+If the uneven-ramp jump detection algorithm is turned on (the default), the ramp fitting
```

Comment:
Can we start with one paragraph ahead of this one motivating jump detection in terms of cosmic ray hits?